src/components/card-section/types.ts

```diff
@@ -11,4 +11,5 @@ export interface FeaturedCardSectionProps {
     content: ContentItem[];
     title?: string;
     className?: string;
+    featuredCardType?: string;
```

Comment:
I think it might be best to explicitly specify the options here. So like
```typescript
featuredCardType: 'middle' | 'large';
```
and then just default that prop to `large`
```typescript
({ content, className, title = 'Featured', featuredCardType = 'large' })
```

It just helps to make more obvious how we can use that prop.